src/app/public/index.ts

```diff
@@ -1 +1,5 @@
 // Export any types that should be included in the root.
+export * from './modules';
```

Comment:
It might be a good idea to list out each file you wish to export in a separate line, instead of pointing to an `index.ts` "barrel" file. This will help weed-out any circular references.